tools/bcdb_mock/models/generated/directory/directory.go

```diff
@@ -96,9 +96,10 @@ func NewTfgridDirectoryNode2() (TfgridDirectoryNode2, error) {
 }
 
 type TfgridDirectoryNodeIface1 struct {
-	Name    string           `bson:"name" json:"name"`
-	Addrs   []schema.IPRange `bson:"addrs" json:"addrs"`
-	Gateway []net.IP         `bson:"gateway" json:"gateway"`
+	Name       string           `bson:"name" json:"name"`
+	Addrs      []schema.IPRange `bson:"addrs" json:"addrs"`
+	Gateway    []net.IP         `bson:"gateway" json:"gateway"`
+	MacAddress net.HardwareAddr `bson:"macaddress" json:"macaddress"`
```

Comment:
You need to add the corresponding field in this file https://github.com/threefoldtech/zos/blob/master/tools/bcdb_mock/models/schema/directory/tfgrid_directory_node_2.toml#L31

You probably need to add something like
```
macaddress = (S) 
```